backend/test/test-setup.ts

```diff
@@ -168,9 +176,13 @@ export class TestSetupBuilder {
    * Returns the database configuration for the test database
    *
    * @param dbName The name of the database to use
+   * @param asAdmin If the database should be connected to as the admin user
```

Comment:
I still don't really understand, what's the purpose of this and why does it hardcode postgres with `root:hedgedoc@.../hedgedoc` as connection?